hawc/services/epa/hero.py

```diff
@@ -28,6 +28,40 @@ def _force_int(val, default=None) -> int | None:
         return default
 
 
+def format_source(content: dict) -> str:
```

Comment:
I added this func to address to TODO you had, since the old HERO API previously gave us a "{journal} {volume}:{pages}" like source string but the new one doesn't, so we can selectively construct it here.  I added a test, I figure we can expand that test overtime if there are some edge cases that aren't handled well by the current implementation.